scripts/spellcheck/dicts/emqx.txt

```diff
@@ -267,3 +267,5 @@ keytab
 jq
 nif
 TDengine
+Clickhouse
```

Comment:
Q: why it has two name format

Reply:
It was to make sure that the spell script would accept the capitalization. I removed it now to test that it goes through event without the capitalization.